### PR TITLE
Navtile

### PR DESCRIPTION
**Pull request by RobZoneNet**
Put in some very basic css for nesting

### CODE REVIEW

File: castle/cms/tiles/navigation.py [hunk 1]
```diff
@@ -76,23 +78,79 @@ def items(self):
...
+                'obj': child[1],
+                'sub_items': []
+            })
+
+            return tree
+
+        brain_listing = [[x.getPath(), x] for x in brains]
+        sorted_brain_listing = sorted(brain_listing, key=lambda x: x[0])
+
+        limit = self.data.get('limit') or 10
```
Comment (tkimnguyen) [inline castle/cms/tiles/navigation.py]: should "10" be defined somewhere more explicitly?

**Comment by obct537** [comment]
It's basically just sorting the brains by their URL's.

File: castle/cms/tiles/navigation.py (hunk 1 above)
Comment (obct537) [inline castle/cms/tiles/navigation.py]: Eh? The "limit" setting is a required option, so it should ideally never need to use that

**Comment by obct537** [comment]
@tkimnguyen Added a little more documentation, still not sure if it's clear.....but it's a tricky algorithm

**Pull request closed without merging**